frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditOptions.test.tsx

```diff
@@ -41,16 +50,22 @@ const renderEditOptions = async <T extends ComponentType.Checkboxes | ComponentT
     ...mockComponent,
     ...componentProps,
   };
-  renderWithProviders(
-    <EditOptions
-      handleComponentChange={handleComponentChange}
-      component={component}
-      renderOptions={renderOptions}
-    />,
-    {
-      queries,
-      queryClient: queryClientMock,
-    },
+  const allQueries = {
+    ...queriesMock,
+    ...queries,
+  };
+
+  render(
```

Comment:
I'll have a look at it, i've done it this way because it's the same way everywhere doReloadPreview is involved but i agree it would look better with renderWithProviders